### PR TITLE
Common healing

### PR DESCRIPTION
**Pull request by jandersson**
I've moved the wound check method from healme and changed its name to reflect what its doing (perceive health) and cherry-picked out my check wounds from necro_heal. The latter method was renamed to check_health to reflect what its doing. I tested the slightly modified healme.lic on my empath and it seems to work fine.

### CODE REVIEW

**Comment by jandersson** [comment]
Some things:
* the hash severity_to_text still needs to be populated. I've been using https://elanthipedia.play.net/mediawiki/index.php/Wounds as a reference.
* check_health does not map nerve damage to skin/nerve as the word skin/nerve does not show up in some of the nerve damage text

File: combat-trainer.lic [hunk 1]
```diff
@@ -152,6 +152,7 @@ class LootProcess
   include DRC
   include DRCI
   include DRCS
+  include DRCH
```
Comment (rpherbig) [inline combat-trainer.lic]: Is this necessary as part of this PR, or is this a line leftover from the necro healing changes?

Comment (jandersson) [inline combat-trainer.lic]: Ah, that is definitely a leftover.

File: healme.lic [hunk 2]
```diff
@@ -133,7 +102,7 @@ class HealMe
     if @settings.empath_healing['HEAL'] && skip_parts.nil?
       spam_heal(@settings.empath_healing['HEAL'].first, @settings.empath_healing['HEAL'][1..-1])
     else
-      wounds = check_wounds
+      wounds = DRCH.check_perc_health
```
Comment (rpherbig) [inline healme.lic]: You should be able to get away with removing `DRCH` here, since it is included above.

File: healme.lic [hunk 3]
```diff
@@ -13,23 +13,9 @@ custom_require.call(%w(common common-arcana events))
 class HealMe
   include DRC
   include DRCA
+  include DRCH
```
Comment (rpherbig) [inline healme.lic]: You'll want to add `common-healing` to the `custom_require` call on line 11.

**Comment by rpherbig** [comment]
What do you need to get the rest of the messaging?

**Comment by jandersson** [comment]
I just need to sit down and do it. All the messaging is on elanthipedia. I also need to figure out how to deal with match text that doesn't have the actual part name in it anywhere, like `minor twitching` for nerve/skin damage

**Comment by rpherbig** [comment]
This looks reasonable to me. What else do you need to do before we merge it @jandersson ?

**Comment by jandersson** [comment]
I can't think of anything, I think that it needs to be tested but for the higher level wound cases. That might be easier when more people are using it

**Pull request merged**